src/main/java/evelyn/task/Task.java

```diff
@@ -0,0 +1,48 @@
+package evelyn.task;
+
+/**
+ * Represents a Task.
+ */
+public class Task {
+    private String description;
+    private boolean isMarked;
+
+    /**
+     * Constructor of a Task object
+     * @param desctiption Description of a Task.
+     * @param isMarked Status of a Task.
+     */
+    public Task (String desctiption, boolean isMarked) {
+        this.description = desctiption;
+        this.isMarked = isMarked;
+    }
+
+    /**
+     * Marks a task as completed.
+     */
+    public void mark() {
+        this.isMarked = true;
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println("    " + this.toString());
+    };
```

Comment:
I noticed there was a couple of places where there was an additional semicolon after the closing curly braces of a method definition, maybe it would be better to remove it?